tensorflow/contrib/mpi_collectives/kernels/mpi_ops.cc

```diff
@@ -13,15 +13,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#ifdef TENSORFLOW_USE_MPI
+// #ifdef TENSORFLOW_USE_MPI
```

Comment:
This is probably going to cause issues for automated build, but I can't figure out how to fix this. Input would be appreciated:

For some reason, Bazel is not adding this preprocessor define to the compile command for these files. It appears that the Bazel target hierarchy contains `tf_additional_mpi_lib_defines` from `tensorflow/core/platform/default/build_config.bzl`, since `tf_kernel_library` target type reduces down to the `cc_library` type. Am I missing something here?

Reply:
Fixed by adding the following to deps: "//tensorflow/core:lib_internal",